zerver/openapi/zulip.yaml

```diff
@@ -3391,6 +3391,12 @@ paths:
                                 Unlike the simpler [realm/update](#realm-update) event format, this
                                 event type supports multiple properties being changed in a
                                 single event.
+
+                                **Changes**: The `email_address_visibility` setting was removed in Zulip 5.0 (feature level 112).
+                                It was replaced by a [user setting](/api/update-settings) with a
+                                [realm user default](api/update-realm-user-settings-defaults), with the encoding of different
+                                values preserved. Clients can support all versions by supporting the current API and treating
+                                every user as having the realm's email address visibility value.
```

Comment:
Though `update_dict` event was not sent on updating realm `email_address_visibility` setting, added this here just because the setting was documented here and also there was some recent discussion to use this event format.

Reply:
Yes, this is correct given the patterns we have here now.